### ISSUE TITLE
Why has sbot CI been passing even though we have no zoloto type stubs?

### ISSUE THREAD

**Issue by kierdavis** [sourcebots/sbot#50]

**Comment by kierdavis** [comment]
```
[nix-shell:~/checkouts/sourcebots/sbot]$ make
poetry run mypy sbot tests 
sbot/vision.py:10: error: Cannot find module named 'zoloto'
sbot/vision.py:10: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
sbot/vision.py:11: error: Cannot find module named 'zoloto.cameras'
sbot/vision.py:14: error: Class cannot subclass 'Camera' (has type 'Any')
sbot/vision.py:14: error: Base type Camera becomes "Any" due to an unfollowed import
```

**Comment by trickeydan** [comment]
This is caused by not having zoloto installed in CI. Solved by #131
